packages/components/src/dropdown-menu/style.scss

```diff
@@ -53,7 +53,7 @@
 
 	.components-menu-item__button,
 	.components-menu-item__button.components-button {
```

Comment:
CSS architecture is very messed up but I doubt we'll refactor this soon-to-be-retired component 😇